apps/web/pages/event-types/index.tsx

```diff
@@ -119,10 +163,10 @@ const EventTypeList = ({ readOnly, types, profile }: EventTypeListProps): JSX.El
             data-disabled={type.$disabled ? 1 : 0}>
             <div
               className={classNames(
-                "flex items-center justify-between hover:bg-neutral-50 ",
+                "hover:bg-neutral-50 flex items-center justify-between ",
                 type.$disabled && "pointer-events-none"
               )}>
-              <div className="group flex w-full items-center justify-between px-4 py-4 hover:bg-neutral-50 sm:px-6">
+              <div className="group hover:bg-neutral-50 flex w-full items-center justify-between px-4 py-4 sm:px-6">
```

Comment:
Let's fix these

Reply:
where do they come from?

Reply:
I have a conflict with two different formatting mechanisms in my computer. Saving a file rearranges the classes but they get marked as errors by other plugin. Fixed.